typeclasses/Rooms.md

```diff
@@ -0,0 +1,52 @@
+# Room typeclass
+### Flags and Sectors
```

Comment:
Everything in this file describes features that are either anti-pattern or not needed.